tests/integration/basic_data/policy-pack/index.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright 2016-2019, Pulumi Corporation.  All rights reserved.
+
+import * as assert from "assert";
+
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+import { PolicyPack } from "@pulumi/policy";
+
+new PolicyPack("basic-data-policy", {
+    policies: [
+        {
+            name: "basic-data-resource-validation",
+            description: "Verifies data during resource validation.",
+            enforcementLevel: "mandatory",
+            validateResource: (_, __) => {
```

Comment:
Nit: IMHO `a, r` or `x, y` would be preferable over `_, __`.

Reply:
Maybe we should have this listed as a standard somewhere? I personally like `_ , __`

Reply:
The reason I used `_, __` is because otherwise the IDE complains:

```
'a' is declared but its value is never read. ts(6133)
```

The code fix suggestion is to prefix the arg names with an underscore, but I actually prefer _just_ using the underscore for unused arguments.

But happy to do whatever other folks prefer.